blake3/go.mod

```diff
@@ -4,9 +4,9 @@ go 1.18
 
 require (
 	github.com/opencontainers/go-digest v0.0.0
-	github.com/zeebo/blake3 v0.1.1
+	github.com/zeebo/blake3 v0.2.3
 )
 
 replace github.com/opencontainers/go-digest => ../
 
-require golang.org/x/sys v0.0.0-20201014080544-cc95f250f6bc // indirect
+require github.com/klauspost/cpuid/v2 v2.0.12 // indirect
```

Comment:
Honestly, this is the only concern, as we'd be adding more non-stdlib dependencies. Alternatively, I could set the minimum version to v0.2.2, which does not have this commit: https://github.com/zeebo/blake3/commit/1e606d3fd6566de23b0c68e4641b401bf586419b

Do we know more context behind the choice to use the klauspost package for this one?

Reply:
⚠️ 👆 I updated the PR to use `v0.2.2` for now; consumers can still decide to pick a newer version, but perhaps someone knows more context about the choice between either of these